run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+kubectl config set-cluster currentCluster --server=\"https://${KUBERNETES_SERVICE_HOST}\"
+kubectl config set-context currentCluster
+kubectl config set-credentials currentCluster --token=\"$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)\"
+kubectl config use-context currentCluster
+
+while true
+do
+  touch /lastloop
+  weave_pod=$(kubectl get pods -n kube-system | grep weave | awk -F ' ' '{print $1}' | shuf | head -1 )
+  echo "Using ${weave_pod}..."
+  [[ -n ${weave_pod} ]]
+  node_regex=$(kubectl get nodes --no-headers -o=custom-columns=NAME:.metadata.name | sed -E "s/ip-(.*)\.ec2.*/\1$/" | paste -d '|' -s -)
+  kubectl exec -i -c weave -n kube-system ${weave_pod} -- /bin/sh -c \
+    'curl -s http://127.0.0.1:6784/status/ipam | \
+    grep unreachable\\!$ | \
+    sed -E "s/.*\(ip-([0-9-]+).*/127.0.0.1:6784\/peer\/\1/" | \
+    sort | \
+    uniq | grep -Ev "('"${node_regex}"')" | \
+    xargs -n 1 curl -sX DELETE'
+  echo "Done."
+  touch /lastloop
```

Comment:
What is the purpose of this?

Reply:
I'm using that file in the liveness check. Wanted to touch it to show we're alive both before and after the long-running operations in case there is some edge case where the delete process takes more than 3 minutes.